### PR TITLE
fix(react-components): Trendcursors ymin max handling and styling fixes

### PR DESCRIPTION
**Pull request by mnischay**
## Overview
Provide an explanation of what the change is

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts [hunk 1]
```diff
@@ -7,36 +7,33 @@ import { calculateSeriesMakers } from '../calculations/calculateSeriesMakers';
...
 const useHandleResize = ({ size, series, graphic, setGraphic, chartRef, visualization }: TrendCursorProps) => {
   const prevSize = useRef(size);
-  // to avoid unnecessary re-rendering
-  if (size.width !== prevSize.current.width || size.height !== prevSize.current.height) {
-    const newG = graphic.map((g) => {
-      // splitting into two separate if block to avoid calculations when not necessary
 
-      // if height has changed, update Y values
-      if (size.height !== prevSize.current.height) {
+  // delaying the updating by 10ms, so that the graph is drawn before reading the values back
```
Comment (tracy-french) [inline packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts:11]: What guarantee do you have that the graph will be drawn first? Are we using the setTimeout to control the event loop?

File: packages/react-components/src/components/chart/trendCursor/seriesYMinMax/useHandleSeriesYMinMax.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        g.timestampInMs,
+        chartRef,
+        visualizationRef.current
+      );
+      g.yAxisMarkerValue = trendCursorsSeriesMakersValue;
+      g.children = updateTrendCursorLineMarkers(g.children, trendCursorsSeriesMakersInPixels);
+      return g;
+    });
+    setGraphic([...newG]);
+  }, [yAxisOptionsString, setGraphic, chartRef, seriesString]);
```
Comment (tracy-french) [inline packages/react-components/src/components/chart/trendCursor/seriesYMinMax/useHandleSeriesYMinMax.ts:53]: Why are we using useEffect to update the state? Is there no event occurring we can call attach a callback to?

Comment (mnischay) [reply packages/react-components/src/components/chart/trendCursor/seriesYMinMax/useHandleSeriesYMinMax.ts:53]: nope, the function inside the useEffect needs to run when the user/api updates the ymin or ymax or series... the change in value itself is an event within the component,,, we can do use the prev reference to achieve this probably, but useEffect made the code look clean.... open for suggestions

File: packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts (hunk 1 above)
Comment (mnischay) [reply packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts:11]: no guarantee, i just need one render cycle... 10ms is an arbitrary time to let one render cycle pass...

Comment (tracy-french) [reply packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts:11]: Seems like an area which may result in bugs in the future. You might be able to get around specifying a time to wait by using https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame.

**Review by tracy-french — APPROVED** [review]

File: packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts (hunk 1 above)
Comment (mnischay) [reply packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts:11]: but requestAnimationFrame lets me know how many times something got render or will render... I cannot skip a frame ? hmmm, let me try.. If there is a alternate solution to using settimeout il take it

**Review by corteggiano — APPROVED** [review]
discussed on call and reviewed ✅

**Pull request merged**